website/content/en/preview/tasks/configuration.md

```diff
@@ -15,7 +15,7 @@ There are two main configuration mechanisms that can be used to configure Karpen
 | Environment Variable | CLI Flag | Description |
 |--|--|--|
 | AWS_DEFAULT_INSTANCE_PROFILE | \-\-aws-default-instance-profile | The default instance profile to use when provisioning nodes in AWS|
-| AWS_ENABLE_POD_ENI | \-\-aws-enable-pod-eni | If true then instances that support pod ENI will report a vpc.amazonaws.com/pod-eni resource (default = false)|
+| AWS_ENABLE_POD_ENI | \-\-aws-enable-pod-eni | If true then instances that support pod ENI will report a vpc.amazonaws.com/pod-eni resource (default = false). **Read the [Pod ENI documentation](../../aws/provisioning/#pod-eni-security-groups-for-pods) before enabling.**|
```

Comment:
Adding to this file manually will get overriden by `docgen` since this file is completely auto-generated. Either add the relevant content to the generation script directly or add it to the CLI help args

Reply:
ah, didn't realize that.  I will take a look.